index.ts

```diff
@@ -1,4 +1,4 @@
-/// <reference path='typings/tsd.d.ts' />
+/// <reference path='typings/definitions.d.ts' />
```

Comment:
It would get our overrides because it `import`'s our local modules, which has the reference to the type definitions :).  At least that is my understanding of what `tsc` is doing.